pkg/application/create_test.go

```diff
@@ -255,14 +265,184 @@ var _ = Describe("Create new application", func() {
 				To: head.Hash(),
 				ExpectedCommits: []gittest.ExpectedCommit{
 					{
-						Message:      "Initial commit\n[skip ci]",
-						ChangedFiles: []string{"./README.md"},
+						Message: "Initial commit\n[skip ci]",
+						ChangedFiles: []string{
+							"README.md",
+							".github/workflows/fast-feedback.yaml",
+							".github/workflows/extended-test.yaml",
+						},
 					},
 				},
 			})
 		})
 		It("pushes all the changes to the remote repository", func() {
 			newAppServerRepo.AssertInSyncWith(newAppLocalRepo)
 		})
+
+		It("returns empty PR url", func() {
+			Expect(createResult.PRUrl).To(Equal(""))
+		})
+
+		It("returns false for monorepo mode", func() {
+			Expect(createResult.MonorepoMode).To(BeFalse())
+		})
+
+	})
+	Context("monorepo mode", Ordered, func() {
```

Comment:
maybe would be good to add one test that fails to create the repo, testing undo operations (like your github mock failing when creating pr or something like this)

Reply:
added a test